Tests/test_elasticsearch.py

```diff
@@ -0,0 +1,148 @@
+# ================================================================================================
+# Tests for Checking the Elasticsearch Layer
+# ================================================================================================
+
+import os
+import json
+import yaml
+import logging
+import argparse
+import requests
+
+
+def init(cluster_name):
```

Comment:
SuggestionL Before getting the ELBs endpoints, we should check that they exist. If they don't, the test should quit.

Reply:
I was thinking of adding a another series of tests to make sure the K8s infra is up. The current PR only addresses the elasticsearch testing of things.

Reply:
The separate tests for the K8S infra sound good, but I still think that we should check that either the ELB endpoints exist or that there is a service running on that DNS:PORT before running everything else. 
As the tests expand and get more complex, they will start taking more time, but if there's nothing running, they will all always fail.

Reply:
Agreed on the fail all tests part. Will add a commit for it.

Reply:
Commit added, lmk if that works.